snippets/dialogs/recently-added-cart-item.liquid

```diff
@@ -40,6 +40,13 @@
                     </div>
                 </div>
             </div>
+            <div ng-show="relatedProducts.item.count">
```

Comment:
Is it necessary to use ng-show? As I know, using ng-if is more preferable.

Reply:
Yes, because we use component output here. If we will use ng-if, it will be cut.